packages/lesswrong/lib/routes.ts

```diff
@@ -356,7 +356,7 @@ addRoute(
   {
     name: 'search',
     path: '/search',
-    componentName: 'SearchPage',
+    componentName: forumTypeSetting.get() === 'EAForum' ? 'SearchPageTabbed' : 'SearchPage',
```

Comment:
This is EAF only, as I didn't think LW was that interested in a tabbed search page. If LW wants to use it, it will take some design work, since a lot of the spacing is different, and we have some extra things like subforums and profile photos. @darkruby501